src/Components/ReviewList.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useState } from "react"
+import { getReviews } from "../utils/api"
+import ReviewCard from "./ReviewCard";
+
+const ReviewList = () => {
+    const [reviews, setReviews] = useState([])
```

Comment:
We also want to add a loading state that will display a loading message if awaiting data from api still. 

Really nice styling and responsive design though!

Once amended, please send another pr :)